src/nwb_benchmarks/command_line_interface.py

```diff
@@ -26,17 +26,25 @@ def main() -> None:
 
     default_asv_machine_file_path = pathlib.Path.home() / ".asv-machine.json"
     if command == "run":
-        process = subprocess.Popen(["asv", "machine", "--yes"], stdout=subprocess.PIPE)
-        process.wait()
+        aws_machine_process = subprocess.Popen(["asv", "machine", "--yes"], stdout=subprocess.PIPE)
+        aws_machine_process.wait()
         customize_asv_machine_file(file_path=default_asv_machine_file_path)
 
         commit_hash = subprocess.check_output(["git", "rev-parse", "--short", "HEAD"]).decode("ascii").strip()
 
         # Save latest environment list from conda (most thorough)
         # subprocess tends to have issues inheriting `conda` entrypoint
         asv_root = pathlib.Path(__file__).parent.parent.parent / ".asv"
+        asv_root.mkdir(exist_ok=True)
+
         raw_environment_info_file_path = asv_root / ".raw_environment_info.txt"
-        os.system(f"conda list > {raw_environment_info_file_path}")
+        environment_info_process = subprocess.Popen(
+            ["conda", "list", ">", raw_environment_info_file_path], stdout=subprocess.PIPE, shell=True
+        )
```

Comment:
@CodyCBakerPhD on my end 

```
 environment_info_process = subprocess.Popen(
            ["conda", "list", ">", raw_environment_info_file_path], stdout=subprocess.PIPE, shell=True
        )
```

never produces a file so that the suite does not run. I changed it to:

```
        outfile = open(raw_environment_info_file_path, "w")
        environment_info_process = subprocess.Popen(
            ["conda list"], stdout=outfile, shell=True
        )
```

which seems to work.

Reply:
That might work even better, thanks

Sure is interesting to see how variable subprocess deployments can be across different people systems, depending on what CLI paths are inherited by child processes